content/attacks/posts/2023-04-14-Bitrue.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2023-04-14
+target-entities:
+- Bitrue
+entity-types:
+  - Custodian
+  - Exchange
+attack-types:
+- Wallet Hack
+title: "Bitrue  Hacked for $23 Million"
+---
+
+## Summary:
+
+On April 14, 2023, cryptocurrency exchange Bitrue was hacked, resulting in the theft of cryptocurrencies worth approximately $23 million. Hackers gained access to the exchange's hot wallets and stole various cryptocurrencies, including ETH, SHIB, QNT, HOT, MATIC, and GALA.
+
+## Attackers:
+
+The identity of the hackers who carried out the attack on Bitrue is unknown.
+
+Bitrue Drainer wallet: [0x1819ede3b8411ebc613f3603813bf42ae09ba5a5](https://etherscan.io/address/0x1819ede3b8411ebc613f3603813bf42ae09ba5a5)
+
+## Losses:
+Bitrue estimated the losses from the hack to be approximately $23 million. The stolen assets included:
+
+* 320 ETH
+* 173,000 QNT
+* 172,55B SHIB
+* 756,15M HOT
+* 310,000 MATIC
+* 46,4M GALA
+
+## Timeline:
+
+- **April 14, 2023:** [Bitrue reported the hack of its hot wallets.](https://twitter.com/BitrueOfficial/status/1646811220543168512)
+- **April 14, 2023:** [Bitrue will temporarily suspend all withdrawals and will reopen withdrawals on 18 April 2023.](https://twitter.com/BitrueOfficial/status/1646811224104136705)
+- **April 14, 2023:** [Bitrue announced that all identified users affected by this incident will receive full compensation.](https://support.bitrue.com/hc/en-001/articles/17538384324121)
+- **April 17, 2023:** [Bitrue resumes normal operations for all tokens in turn except $QNT, $MATIC, and $HOT.](https://support.bitrue.com/hc/en-001/articles/17617922874649-Update-on-Bitrue-s-Security-Breach-17-04-2023-)
+
+## Security Failure Causes:
+
+- **Wallet System Vulnerability:** Flaws in Bitrue's hot wallet system have led to unauthorized access and transfers of funds.
```

Comment:
If it's not an official failure reason, mention that it's most possible cause and detalize. Examples can be found on other `Wallet Hack` type articles

Reply:
No details. The only information that the hot wallet was hacked